### DISCUSSION TITLE
Next.js app cannot be deployed on Vercel

### DISCUSSION THREAD

**Discussion by georgegeo1** [vercel/next.js#28402] [Help]
A "Failed to compile" error due to "Module Not Found" keeps showing up. I read that this error is a case sensitive error, since the compiler tries to find "../components/component1" while the route is "../components/Components1". Should I rename the files and use lowercase letters? Does this make any sense? Any suggestion is welcome.

**Comment by oliverloops** [comment]
Hi @georgegeo1, when you write a component file is actually a standard to start with an uppercase. Maybe in this case verify if this is not a typo on your file name or a wrong route in your imported components. I said this just in case something is wrong, I've never faced this problem before and I used to name my component files with an uppercase at the beginning.
